### DISCUSSION TITLE
Unable to export?

### DISCUSSION THREAD

**Discussion by aecatec** [Tyrrrz/DiscordChatExporter#1181] [Q&A]
Hi! I just tried running the export command and I get this message: 

`export: not valid in this context: token`

I have double checked my token is correct, and I can't find this issue anywhere. I am on MacOS. This is my first time posting something like this, so please let me know if any more information is needed!

**Comment by Tyrrrz** [comment]
What is the command you're running?

**Reply by aecatec** [reply]
Sorry for the delay! I'm trying to run the export command.

**Reply by Tyrrrz** [reply]
I can see that, can you post the full command line input? You can obfuscate any sensitive data within it.

**Reply by aecatec** [reply]
Oh, of course, got it! 

`export -t [token]  -c [channelid]`

I have tried this with and without quotes for the token, and I get the same error.

**Reply by Tyrrrz** [reply]
You have to prepend it with `dotnet DiscordChatExporter.dll` on macOS
